thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/spec/WoWStdRuleDetectorSpec.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.pinot.thirdeye.detection.spec;
+
+import org.apache.pinot.thirdeye.dataframe.util.MetricSlice;
+import org.apache.pinot.thirdeye.detection.Pattern;
+
+
+public class WoWStdRuleDetectorSpec extends AbstractSpec {
+  private String monitoringGranularity = MetricSlice.NATIVE_GRANULARITY.toAggregationGranularityString(); // use native granularity by default
+  private int lookback = 52;
+  private double sensitivity = 1.5;
```

Comment:
We should make the sensitivity from 0-10. Higher value means more sensitive.

Reply:
I think the 0-10 scale makes it very hard to map how many total sigmas we are trying to use or sigmas on each side. Typically the "sensitivity" here should be between 0.5 and 1.5 since the number is for sigmas on each side. The interval length is matching to 0-10, but then it would have a base value of 0.5, which is a little confusing. Maybe changing the name to "sigmas" instead of "sensitivity" to avoid confusion?

Reply:
I change it to sigma and added validation part to make sure sigma should be between 0-1.5 Thanks!

Reply:
I think you can keep sigma. However the motivation of introducing 0-10 scale is to give a universal parameter setting across all anomaly detection functions. There are several benefits:
1. Easy to understand. Users don't need to understand the methodology behind it. It could be p-value or sigma or others.
2. Easy to switch between rules w/o changing parameters. E.g, user can set a sensitivity of 5 to expect a moderate sensitivity for all rules.
I strongly encourage you to generate a mapping between sensitivity and sigma and expose sensitivity to user. If you want to expose sigma for users to fine control the model (like p-value) it is fine.

Reply:
Thanks for the suggestion. I changed the spec and detection to use sensitivity.